shopinvader_api_payment_provider_custom/schemas/transaction_processing_values.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2024 ACSONE SA (https://acsone.eu).
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+
+from odoo.addons.shopinvader_api_payment.schemas import (
+    TransactionProcessingValues as BaseTransactionProcessingValues,
+)
+
+
+class TransactionProcessingValues(BaseTransactionProcessingValues, extends=True):
+    payment_info_form_html: str | None = None
```

Comment:
@sbidoul Thibault asked for a new field, different from `redirect_form_html`.
Because the form he gets in `redirect_form_html` is directly submitted, hence if he wants to follow the same behavior for all payment providers, the customer won't see the payment info he displays, as he will directly call our `custom/pending` route.